torch/lib/c10d/Utils.cpp

```diff
@@ -203,7 +229,7 @@ int connect(
   struct ::addrinfo hints, *res = NULL;
   std::memset(&hints, 0x00, sizeof(hints));
   hints.ai_flags = AI_NUMERICSERV; // specifies that port (service) is numeric
-  hints.ai_family = AF_UNSPEC; // either IPv4 or IPv6
+  hints.ai_family = AF_INET;//AF_UNSPEC;//AF_INET; // either IPv4 or IPv6
```

Comment:
I am not familiar with this. Any reason for this change?

cc @lw for help.

Reply:
Will change it back, this is for my local box.